src/MonologCreator/Processor/RequestId.php

```diff
@@ -133,11 +108,11 @@ protected function isCallable($callable)
      * xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx
      * where x is 0-9, A-F and y is 8-9, A-B.
      *
-     * @param $data     string  random bytes.
+     * @param string $data - random bytes.
      *
-     * @return string           valid, formatted UUIDv4
+     * @return string - valid, formatted UUIDv4
```

Comment:
`@param` and `@return` can be removed, right?

Reply:
right, I've missed to remove these comments. I've fixed them.